### PR TITLE
Fixes to group join examples parsing

### PR DESCRIPTION
**Pull request by Cohee1207**
## Checklist:

- [X] I have read the [Contributing guidelines](https://github.com/SillyTavern/SillyTavern/blob/release/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cloak1505** [comment]
image

Nice, {{char}}-only example messages are no longer missing. However, the active char will have its name prefixed to all char examples. Funnily, quintuplets personalities seem to remain intact in CC (tells model who to reply as).

~~Negatively affects TC, they will immediately output a reply as another character despite a name prefilled.~~ EDIT: I'm cycling through Include Names and I don't see any extra names attached. TC works normally.

Not affected by Names Behavior, or unchecked system prompt in the case of Google AI Studio which just sends the same thing as user.

Edit: OpenRouter appears normal. The same thing as appears in Prompt Inspector extension, containing OpenAI "name":"example_user/assistant" key-value pairs.

Basically, OAI and TC okay, things like Claude, Cohere, and Gemini bugged with active char prefixing.

**Comment by Cohee1207** [comment]
@cloak1505 I had to rewrite all prompt converters to make them aware of group names in example messages. I hope nothing broke with that, lol.

**Comment by cloak1505** [comment]
Boom. Fixed.

I tested Claude, Cohere, Gemini, OpenRouter CC/TC, and KoboldCpp, and do not see any mistakes whatsoever.
